exporter/awsxrayexporter/awsxray_test.go

```diff
@@ -54,31 +74,49 @@ func BenchmarkForTracesExporter(b *testing.B) {
 }
 
 func initializeTracesExporter() component.TracesExporter {
-	os.Setenv("AWS_ACCESS_KEY_ID", "AKIASSWVJUY4PZXXXXXX")
-	os.Setenv("AWS_SECRET_ACCESS_KEY", "XYrudg2H87u+ADAAq19Wqx3D41a09RsTXXXXXXXX")
-	os.Setenv("AWS_DEFAULT_REGION", "us-east-1")
-	os.Setenv("AWS_REGION", "us-east-1")
-	factory := NewFactory()
-	config := factory.CreateDefaultConfig()
-	config.(*Config).Region = "us-east-1"
-	config.(*Config).LocalMode = true
+	exporterConfig := generateConfig()
 	mconn := new(awsutil.Conn)
-	traceExporter, err := newTracesExporter(config, componenttest.NewNopExporterCreateSettings(), mconn)
+	traceExporter, err := newTracesExporter(exporterConfig, componenttest.NewNopExporterCreateSettings(), mconn)
 	if err != nil {
 		panic(err)
 	}
 	return traceExporter
 }
 
+func generateConfig() config.Exporter {
+	os.Setenv("AWS_ACCESS_KEY_ID", "AKIASSWVJUY4PZXXXXXX")
+	os.Setenv("AWS_SECRET_ACCESS_KEY", "XYrudg2H87u+ADAAq19Wqx3D41a09RsTXXXXXXXX")
+	os.Setenv("AWS_DEFAULT_REGION", "us-east-1")
+	os.Setenv("AWS_REGION", "us-east-1")
+	factory := NewFactory()
+	exporterConfig := factory.CreateDefaultConfig()
+	exporterConfig.(*Config).Region = "us-east-1"
+	exporterConfig.(*Config).LocalMode = true
+	return exporterConfig
+}
+
 func constructSpanData() pdata.Traces {
 	resource := constructResource()
 
 	traces := pdata.NewTraces()
 	rspans := traces.ResourceSpans().AppendEmpty()
 	resource.CopyTo(rspans.Resource())
 	ispans := rspans.InstrumentationLibrarySpans().AppendEmpty()
-	constructHTTPClientSpan().CopyTo(ispans.Spans().AppendEmpty())
-	constructHTTPServerSpan().CopyTo(ispans.Spans().AppendEmpty())
+	constructHTTPClientSpan(newTraceID()).CopyTo(ispans.Spans().AppendEmpty())
+	constructHTTPServerSpan(newTraceID()).CopyTo(ispans.Spans().AppendEmpty())
+	return traces
+}
+
+func constructMixedSpanData() pdata.Traces {
```

Comment:
I think it's a bit hard to understand `MixedSpanData`. I had a hard time describing it when I was writing the issue too.

Is it possible for this function to mirror `constructSpanData` more closely, but ONLY create Traces with non-X-Ray compliant Trace IDs?

Then you could do something like

```go
good_td := constructSpanData()
bad_td := constructSpanDataWithW3CFormatTraceIds()
td := [good_td, bad_td]
```

Sorry if syntax is off!